pkg/crypto/crypto.go

```diff
@@ -0,0 +1,104 @@
+package crypto
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/armor"
+)
+
+// EncryptUsername returns the encrypted value to be used as a username annotation
+func EncryptUsername(username string, key []byte) (string, error) {
+	encryptedUsername, err := encrypt(username, key)
+	if err != nil {
+		return "", err
+	}
+	// Make encrypted data compatible as JSON Patch request body.
+	// Newline characters are marked with text placeholder in order to be easily reversed during decryption
+	return strings.Replace(encryptedUsername, "\n", "<newline>", -1), nil
```

Comment:
Now that I understand what's going on, we should make `<newline>` a constant then.